File: .vscode/IFAL/Python/[#15].py
'''[#15]  Verifique se um número é palíndromo. Um número é palíndromo se continua o mesmo caso seus 
dígitos sejam invertidos. Exemplos: 454, 10501.'''

a = int(input("Digite um primeiro número: "))
b = int(input("Um segundo: "))
c = int(input("Um terceiro: "))
d = int(input("Um quarto número: "))
e = int(input("E o quinto número: "))

if a == e:
    if b == d:
        print ("\nEsse número é um palíndromo")
        
    else:
        print ("\nEsse número não é palíndromo")
        
else:
    print ("\nEsse número não é um palíndromo")

File: programas_C/[#15].c
/*[#15]  Verifique se um número é palíndromo. Um número é palíndromo se continua o mesmo caso seus 
dígitos sejam invertidos. Exemplos: 454, 10501.*/

#include <stdio.h>
int main() {
    int a, b, c, d, e;

    printf("Digite um primeiro número: ");
    scanf("%d", &a);
    printf("Um segundo: ");
    scanf("%d", &b);
    printf("Um terceiro: ");
    scanf("%d", &c);
    printf("Um quarto número: ");
    scanf("%d", &d);
    printf("E o quinto número: ");
    scanf("%d", &e);

    if (a == e) {

        if (b == d) {
            printf("\nEsse número é um palíndromo\n");
        } else {
            printf("\nEsse número não é palíndromo\n");
        }

    } else {
        printf("\nEsse número não é um palíndromo\n");
    }
    return 0;
}